docs/UserGuide.md

```diff
@@ -12,6 +12,7 @@ It allows for faster and more effective student management.
 **Table of Contents**
 * [Quick start](#quick-start)
 * [Features](#features)
+  * [Open help panel: `help`](#viewing-help--help)
```

Comment:
Should help be placed below together with `clear` and `exit` instead?

Reply:
Hmmmm I followed https://nus-cs2103-ay2021s2.github.io/tp/UserGuide.html ah. I think the reason why they did that is that it is more sequential ah. A person who starts must know how to "get help" before even starting to learn about the commands. After trying out all the commands, he / she can then clear the application and then "start" actually using their own data for the application. Thereafter they can exit the application. This seems to be quite a logic flow. What do you think?